astro_2.0/components/AppHeader/components/SearchBar/SearchBar.module.scss

```diff
@@ -93,3 +94,20 @@ $search-max-width: 768;
     color: var(--color-brand-purple);
   }
 }
+
+.hint {
+  @extend %caption2;
+  background: var(--color-neutral-10);
+  border-radius: 8px;
+  box-shadow: 0 8.89583px 30px rgba(0, 0, 0, 0.14);
```

Comment:
I think 9px is ok for shadow instead of 8.89583px
And we have variable --color-black-opacity-14 for this rgba color.